recipes/openssl/openssl_1.0.2g.oe

```diff
@@ -1,8 +1,5 @@
 require openssl.inc
 
-# Parallel make is simply broken for now.
-PARALLEL_MAKE = ""
```

Comment:
Arch does not do anything special:
https://projects.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/openssl
Yocto however have a patch:
http://git.yoctoproject.org/cgit.cgi/poky/plain/meta/recipes-connectivity/openssl/openssl/parallel.patch
I works on my builder with -j8 :-)